src/components/Api.js

```diff
@@ -0,0 +1,146 @@
+export default class Api {
+  constructor(options) {
+    this.baseUrl = options.baseUrl;
+    this.header = options.header;
+  }
+
+  getInitialCards() {
+    return fetch("https://around-api.en.tripleten-services.com/v1/cards", {
+      headers: {
+        authorization: "6bc22e8b-f205-4c04-b4ee-f39ef81e1654",
+        method: "GET",
+      },
+    })
+      .then((res) => res.json())
+      .then((result) => {
+        return result;
+      })
+      .catch((err) => {
+        console.log(err);
+      });
+  }
+
+  renderUserInfo(userData) {
+    const userName = document.querySelector(".profile__title");
+    const userDescription = document.querySelector(".profile__subtitle");
+    const userAvatar = document.querySelector(".profile__avatar");
```

Comment:
`Api.js` is responsible only for defining the methods and checking responses. It can't know about `DOM` elements